### PR TITLE
I added new columns to address table

### PR DESCRIPTION
**Pull request by Zingerd**

### CODE REVIEW

**Review by Max877866 — APPROVED** [review]

**Review by KuzbytMaksym — APPROVED** [review]

**Review by Natalia1910 — APPROVED** [review]

File: dao/src/main/java/greencity/entity/user/ubs/Address.java [hunk 1]
```diff
@@ -69,4 +69,16 @@
 
     @Embedded
     private Coordinates coordinates;
+
+    @Column(name = "city_en", nullable = false)
```
Comment (VadymMakitra) [inline dao/src/main/java/greencity/entity/user/ubs/Address.java:73]: Is it really need to declare in annotation ?

File: dao/src/main/java/greencity/entity/user/ubs/Address.java [hunk 2]
```diff
@@ -69,4 +69,16 @@
...
+    private String cityEn;
+
+    @Column(name = "region_en", nullable = false)
+    private String regionEn;
```
Comment (VadymMakitra) [inline dao/src/main/java/greencity/entity/user/ubs/Address.java:77]: What mean prefix En ? Maybe better to rename field for example region ?

File: service-api/src/main/java/greencity/dto/AddressDto.java [hunk 3]
```diff
@@ -43,4 +43,15 @@
     private Coordinates coordinates;
 
     private Boolean actual;
+    @NotBlank
+    @Length(max = 30)
+    private String cityEn;
```
Comment (VadymMakitra) [inline service-api/src/main/java/greencity/dto/AddressDto.java:48]: Same with entity

**Review by VadymMakitra — COMMENTED** [review]
Please resolve comments in next pr;

File: dao/src/main/java/greencity/entity/user/ubs/Address.java (hunk 1 above)
Comment (thesvider) [reply dao/src/main/java/greencity/entity/user/ubs/Address.java:73]: no need as the filed name is the same

File: service-api/src/main/java/greencity/dto/OrderAddressDtoRequest.java [hunk 4]
```diff
@@ -48,4 +48,16 @@
     private Coordinates coordinates;
 
     private Boolean actual;
+    @Length(max = 30)
+    @Pattern(regexp = "[ЁёІіЇїҐґЄєА-Яа-яA-Za-z-\\s'.]{3,30}")
+    private String cityEn;
```
Comment (thesvider) [inline service-api/src/main/java/greencity/dto/OrderAddressDtoRequest.java:53]: this is supposed to be english version of city? why you're accepting ЁёІіЇїҐґЄєА-Яа-яA then?

File: dao/src/main/resources/db/changelog/logs/ch-add-columns-address-Boiarchuk.xml [hunk 5]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet id="Max-101" author="Max Boiarchuk">
```
Comment (thesvider) [inline dao/src/main/resources/db/changelog/logs/ch-add-columns-address-Boiarchuk.xml:5]: why do you create separate changeset for each column? create just one changeset that adds all columns
